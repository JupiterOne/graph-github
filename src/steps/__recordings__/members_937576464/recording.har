{
  "log": {
    "_recordingName": "members",
    "creator": {
      "comment": "persister:JupiterOneIntegationFSPersister",
      "name": "Polly.JS",
      "version": "5.1.1"
    },
    "entries": [
      {
        "_id": "06d6593fc4a07c98683e9f0972b1c5e0",
        "_order": 0,
        "cache": {},
        "request": {
          "bodySize": 0,
          "cookies": [],
          "headers": [
            {
              "_fromType": "array",
              "name": "accept",
              "value": "application/vnd.github.machine-man-preview+json"
            },
            {
              "_fromType": "array",
              "name": "user-agent",
              "value": "jupiter-integration-github octokit-rest.js/18.12.0 octokit-core.js/3.5.1 Node.js/16.14.2 (darwin; x64)"
            },
            {
              "_fromType": "array",
              "name": "authorization",
              "value": "[REDACTED]"
            },
            {
              "_fromType": "array",
              "name": "content-length",
              "value": "0"
            },
            {
              "_fromType": "array",
              "name": "accept-encoding",
              "value": "gzip,deflate"
            },
            {
              "_fromType": "array",
              "name": "connection",
              "value": "close"
            },
            {
              "name": "host",
              "value": "api.github.com"
            }
          ],
          "headersSize": 813,
          "httpVersion": "HTTP/1.1",
          "method": "POST",
          "queryString": [],
          "url": "https://api.github.com/app/installations/23522163/access_tokens"
        },
        "response": {
          "bodySize": 102,
          "content": {
            "mimeType": "application/json; charset=utf-8",
            "size": 102,
            "text": "{\"message\":\"A JSON web token could not be decoded\",\"documentation_url\":\"https://docs.github.com/rest\",\"token\":\"[REDACTED]\",\"expires_at\":\"2050-12-31T18:09:20Z\"}"
          },
          "cookies": [],
          "headers": [
            {
              "name": "server",
              "value": "GitHub.com"
            },
            {
              "name": "date",
              "value": "Fri, 02 Sep 2022 15:59:41 GMT"
            },
            {
              "name": "content-type",
              "value": "application/json; charset=utf-8"
            },
            {
              "name": "content-length",
              "value": "102"
            },
            {
              "name": "x-github-media-type",
              "value": "github.v3; param=machine-man-preview; format=json"
            },
            {
              "name": "access-control-expose-headers",
              "value": "ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset"
            },
            {
              "name": "access-control-allow-origin",
              "value": "*"
            },
            {
              "name": "strict-transport-security",
              "value": "max-age=31536000; includeSubdomains; preload"
            },
            {
              "name": "x-frame-options",
              "value": "deny"
            },
            {
              "name": "x-content-type-options",
              "value": "nosniff"
            },
            {
              "name": "x-xss-protection",
              "value": "0"
            },
            {
              "name": "referrer-policy",
              "value": "origin-when-cross-origin, strict-origin-when-cross-origin"
            },
            {
              "name": "content-security-policy",
              "value": "default-src 'none'"
            },
            {
              "name": "vary",
              "value": "Accept-Encoding, Accept, X-Requested-With"
            },
            {
              "name": "x-github-request-id",
              "value": "F4B9:689B:57D06:D196E:6312286D"
            },
            {
              "name": "connection",
              "value": "close"
            }
          ],
          "headersSize": 939,
          "httpVersion": "HTTP/1.1",
          "redirectURL": "",
          "status": 401,
          "statusText": "Unauthorized"
        },
        "startedDateTime": "2022-09-02T15:59:40.909Z",
        "time": 160,
        "timings": {
          "blocked": -1,
          "connect": -1,
          "dns": -1,
          "receive": 0,
          "send": 0,
          "ssl": -1,
          "wait": 160
        }
      }
    ],
    "pages": [],
    "version": "1.2"
  }
}
