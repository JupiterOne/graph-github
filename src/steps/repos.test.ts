jest.setTimeout(20000);

import {
  Recording,
  executeStepWithDependencies,
} from '@jupiterone/integration-sdk-testing';
import { buildStepTestConfig } from '../../test/config';
import { setupGithubRecording } from '../../test/recording';
import { Steps } from '../constants';

let recording: Recording;
afterEach(async () => {
  await recording.stop();
});

test('fetchRepos exec handler', async () => {
  recording = setupGithubRecording({
    directory: __dirname,
    name: 'repos',
  });

  const stepConfig = buildStepTestConfig(Steps.FETCH_REPOS);
  const stepResults = await executeStepWithDependencies(stepConfig);

  expect(stepResults).toMatchStepMetadata(stepConfig);
});
