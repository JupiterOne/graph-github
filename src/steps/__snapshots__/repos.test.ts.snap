// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fetchRepos exec handler 1`] = `
Object {
  "collectedEntities": Array [
    Object {
      "_class": Array [
        "CodeRepo",
      ],
      "_key": "R_kgDOG5ypUQ",
      "_rawData": Array [
        Object {
          "name": "default",
          "rawData": Object {
            "autoMergeAllowed": false,
            "createdAt": "2022-02-24T18:04:58Z",
            "databaseId": 463251793,
            "deleteBranchOnMerge": false,
            "description": null,
            "forkCount": 0,
            "forkingAllowed": false,
            "homepageUrl": null,
            "id": "R_kgDOG5ypUQ",
            "isArchived": false,
            "isDisabled": false,
            "isEmpty": false,
            "isFork": false,
            "isInOrganization": true,
            "isLocked": false,
            "isMirror": false,
            "isPrivate": true,
            "isSecurityPolicyEnabled": false,
            "isTemplate": false,
            "isUserConfigurationRepository": false,
            "lockReason": null,
            "mergeCommitAllowed": true,
            "name": "scaling-lamp",
            "nameWithOwner": "j1-ingest/scaling-lamp",
            "pushedAt": "2022-08-25T20:56:16Z",
            "rebaseMergeAllowed": true,
            "updatedAt": "2022-03-11T16:06:51Z",
            "url": "https://github.com/j1-ingest/scaling-lamp",
          },
        },
      ],
      "_type": "github_repo",
      "archived": false,
      "autoMergeAllowed": false,
      "createdOn": 1645725898000,
      "databaseId": 463251793,
      "deleteBranchOnMerge": false,
      "description": "",
      "disabled": false,
      "displayName": "scaling-lamp",
      "empty": false,
      "fork": false,
      "forkCount": 0,
      "forkingAllowed": false,
      "fullName": "j1-ingest/scaling-lamp",
      "homepageUrl": "",
      "inOrganization": true,
      "lockReason": "",
      "locked": false,
      "mergeCommitAllowed": true,
      "mirror": false,
      "name": "scaling-lamp",
      "node": "R_kgDOG5ypUQ",
      "owner": "j1-ingest",
      "public": false,
      "pushedOn": 1661460976000,
      "rebaseMergeAllowed": true,
      "securityPolicyEnabled": false,
      "template": false,
      "updatedOn": 1647014811000,
      "userConfigurationRepository": false,
      "webLink": "https://github.com/j1-ingest/scaling-lamp",
    },
    Object {
      "_class": Array [
        "CodeRepo",
      ],
      "_key": "R_kgDOG-zMhA",
      "_rawData": Array [
        Object {
          "name": "default",
          "rawData": Object {
            "autoMergeAllowed": false,
            "createdAt": "2022-03-10T20:40:46Z",
            "databaseId": 468503684,
            "deleteBranchOnMerge": false,
            "description": "Testing for data ingestion",
            "forkCount": 1,
            "forkingAllowed": true,
            "homepageUrl": null,
            "id": "R_kgDOG-zMhA",
            "isArchived": false,
            "isDisabled": false,
            "isEmpty": false,
            "isFork": false,
            "isInOrganization": true,
            "isLocked": false,
            "isMirror": false,
            "isPrivate": false,
            "isSecurityPolicyEnabled": false,
            "isTemplate": false,
            "isUserConfigurationRepository": false,
            "lockReason": null,
            "mergeCommitAllowed": false,
            "name": "reimagined-barnacle",
            "nameWithOwner": "j1-ingest/reimagined-barnacle",
            "pushedAt": "2022-08-25T19:17:41Z",
            "rebaseMergeAllowed": false,
            "updatedAt": "2022-03-10T20:40:46Z",
            "url": "https://github.com/j1-ingest/reimagined-barnacle",
          },
        },
      ],
      "_type": "github_repo",
      "archived": false,
      "autoMergeAllowed": false,
      "createdOn": 1646944846000,
      "databaseId": 468503684,
      "deleteBranchOnMerge": false,
      "description": "Testing for data ingestion",
      "disabled": false,
      "displayName": "reimagined-barnacle",
      "empty": false,
      "fork": false,
      "forkCount": 1,
      "forkingAllowed": true,
      "fullName": "j1-ingest/reimagined-barnacle",
      "homepageUrl": "",
      "inOrganization": true,
      "lockReason": "",
      "locked": false,
      "mergeCommitAllowed": false,
      "mirror": false,
      "name": "reimagined-barnacle",
      "node": "R_kgDOG-zMhA",
      "owner": "j1-ingest",
      "public": true,
      "pushedOn": 1661455061000,
      "rebaseMergeAllowed": false,
      "securityPolicyEnabled": false,
      "template": false,
      "updatedOn": 1646944846000,
      "userConfigurationRepository": false,
      "webLink": "https://github.com/j1-ingest/reimagined-barnacle",
    },
    Object {
      "_class": Array [
        "CodeRepo",
      ],
      "_key": "R_kgDOG_MJdQ",
      "_rawData": Array [
        Object {
          "name": "default",
          "rawData": Object {
            "autoMergeAllowed": false,
            "createdAt": "2022-03-11T22:04:54Z",
            "databaseId": 468912501,
            "deleteBranchOnMerge": false,
            "description": "Team nesting testing",
            "forkCount": 0,
            "forkingAllowed": false,
            "homepageUrl": null,
            "id": "R_kgDOG_MJdQ",
            "isArchived": false,
            "isDisabled": false,
            "isEmpty": false,
            "isFork": false,
            "isInOrganization": true,
            "isLocked": false,
            "isMirror": false,
            "isPrivate": true,
            "isSecurityPolicyEnabled": false,
            "isTemplate": false,
            "isUserConfigurationRepository": false,
            "lockReason": null,
            "mergeCommitAllowed": true,
            "name": "friendly-spork",
            "nameWithOwner": "j1-ingest/friendly-spork",
            "pushedAt": "2022-06-23T13:40:26Z",
            "rebaseMergeAllowed": true,
            "updatedAt": "2022-03-11T22:04:54Z",
            "url": "https://github.com/j1-ingest/friendly-spork",
          },
        },
      ],
      "_type": "github_repo",
      "archived": false,
      "autoMergeAllowed": false,
      "createdOn": 1647036294000,
      "databaseId": 468912501,
      "deleteBranchOnMerge": false,
      "description": "Team nesting testing",
      "disabled": false,
      "displayName": "friendly-spork",
      "empty": false,
      "fork": false,
      "forkCount": 0,
      "forkingAllowed": false,
      "fullName": "j1-ingest/friendly-spork",
      "homepageUrl": "",
      "inOrganization": true,
      "lockReason": "",
      "locked": false,
      "mergeCommitAllowed": true,
      "mirror": false,
      "name": "friendly-spork",
      "node": "R_kgDOG_MJdQ",
      "owner": "j1-ingest",
      "public": false,
      "pushedOn": 1655991626000,
      "rebaseMergeAllowed": true,
      "securityPolicyEnabled": false,
      "template": false,
      "updatedOn": 1647036294000,
      "userConfigurationRepository": false,
      "webLink": "https://github.com/j1-ingest/friendly-spork",
    },
    Object {
      "_class": Array [
        "CodeRepo",
      ],
      "_key": "R_kgDOG_MLeg",
      "_rawData": Array [
        Object {
          "name": "default",
          "rawData": Object {
            "autoMergeAllowed": false,
            "createdAt": "2022-03-11T22:07:16Z",
            "databaseId": 468913018,
            "deleteBranchOnMerge": false,
            "description": "Test Repo for ingestion",
            "forkCount": 0,
            "forkingAllowed": false,
            "homepageUrl": null,
            "id": "R_kgDOG_MLeg",
            "isArchived": false,
            "isDisabled": false,
            "isEmpty": false,
            "isFork": false,
            "isInOrganization": true,
            "isLocked": false,
            "isMirror": false,
            "isPrivate": true,
            "isSecurityPolicyEnabled": false,
            "isTemplate": false,
            "isUserConfigurationRepository": false,
            "lockReason": null,
            "mergeCommitAllowed": true,
            "name": "automatic-parakeet",
            "nameWithOwner": "j1-ingest/automatic-parakeet",
            "pushedAt": "2022-03-11T22:07:17Z",
            "rebaseMergeAllowed": true,
            "updatedAt": "2022-03-11T22:07:16Z",
            "url": "https://github.com/j1-ingest/automatic-parakeet",
          },
        },
      ],
      "_type": "github_repo",
      "archived": false,
      "autoMergeAllowed": false,
      "createdOn": 1647036436000,
      "databaseId": 468913018,
      "deleteBranchOnMerge": false,
      "description": "Test Repo for ingestion",
      "disabled": false,
      "displayName": "automatic-parakeet",
      "empty": false,
      "fork": false,
      "forkCount": 0,
      "forkingAllowed": false,
      "fullName": "j1-ingest/automatic-parakeet",
      "homepageUrl": "",
      "inOrganization": true,
      "lockReason": "",
      "locked": false,
      "mergeCommitAllowed": true,
      "mirror": false,
      "name": "automatic-parakeet",
      "node": "R_kgDOG_MLeg",
      "owner": "j1-ingest",
      "public": false,
      "pushedOn": 1647036437000,
      "rebaseMergeAllowed": true,
      "securityPolicyEnabled": false,
      "template": false,
      "updatedOn": 1647036436000,
      "userConfigurationRepository": false,
      "webLink": "https://github.com/j1-ingest/automatic-parakeet",
    },
    Object {
      "_class": Array [
        "CodeRepo",
      ],
      "_key": "R_kgDOHDREFw",
      "_rawData": Array [
        Object {
          "name": "default",
          "rawData": Object {
            "autoMergeAllowed": false,
            "createdAt": "2022-03-23T12:54:36Z",
            "databaseId": 473187351,
            "deleteBranchOnMerge": false,
            "description": "A test for locking down a repo (and PRs?)",
            "forkCount": 0,
            "forkingAllowed": false,
            "homepageUrl": null,
            "id": "R_kgDOHDREFw",
            "isArchived": false,
            "isDisabled": false,
            "isEmpty": false,
            "isFork": false,
            "isInOrganization": true,
            "isLocked": false,
            "isMirror": false,
            "isPrivate": true,
            "isSecurityPolicyEnabled": false,
            "isTemplate": false,
            "isUserConfigurationRepository": false,
            "lockReason": null,
            "mergeCommitAllowed": true,
            "name": "didactic-octo-parakeet-LOCKED",
            "nameWithOwner": "j1-ingest/didactic-octo-parakeet-LOCKED",
            "pushedAt": "2022-05-19T18:46:36Z",
            "rebaseMergeAllowed": true,
            "updatedAt": "2022-03-23T14:47:53Z",
            "url": "https://github.com/j1-ingest/didactic-octo-parakeet-LOCKED",
          },
        },
      ],
      "_type": "github_repo",
      "archived": false,
      "autoMergeAllowed": false,
      "createdOn": 1648040076000,
      "databaseId": 473187351,
      "deleteBranchOnMerge": false,
      "description": "A test for locking down a repo (and PRs?)",
      "disabled": false,
      "displayName": "didactic-octo-parakeet-LOCKED",
      "empty": false,
      "fork": false,
      "forkCount": 0,
      "forkingAllowed": false,
      "fullName": "j1-ingest/didactic-octo-parakeet-locked",
      "homepageUrl": "",
      "inOrganization": true,
      "lockReason": "",
      "locked": false,
      "mergeCommitAllowed": true,
      "mirror": false,
      "name": "didactic-octo-parakeet-LOCKED",
      "node": "R_kgDOHDREFw",
      "owner": "j1-ingest",
      "public": false,
      "pushedOn": 1652985996000,
      "rebaseMergeAllowed": true,
      "securityPolicyEnabled": false,
      "template": false,
      "updatedOn": 1648046873000,
      "userConfigurationRepository": false,
      "webLink": "https://github.com/j1-ingest/didactic-octo-parakeet-LOCKED",
    },
    Object {
      "_class": Array [
        "CodeRepo",
      ],
      "_key": "R_kgDOHDYIqQ",
      "_rawData": Array [
        Object {
          "name": "default",
          "rawData": Object {
            "autoMergeAllowed": false,
            "createdAt": "2022-03-23T18:03:51Z",
            "databaseId": 473303209,
            "deleteBranchOnMerge": false,
            "description": "Testing for archived repos",
            "forkCount": 0,
            "forkingAllowed": false,
            "homepageUrl": null,
            "id": "R_kgDOHDYIqQ",
            "isArchived": false,
            "isDisabled": false,
            "isEmpty": false,
            "isFork": false,
            "isInOrganization": true,
            "isLocked": false,
            "isMirror": false,
            "isPrivate": true,
            "isSecurityPolicyEnabled": false,
            "isTemplate": false,
            "isUserConfigurationRepository": false,
            "lockReason": null,
            "mergeCommitAllowed": true,
            "name": "expert-spoon-ARCHIVED",
            "nameWithOwner": "j1-ingest/expert-spoon-ARCHIVED",
            "pushedAt": "2022-03-23T18:04:18Z",
            "rebaseMergeAllowed": true,
            "updatedAt": "2022-03-23T18:03:51Z",
            "url": "https://github.com/j1-ingest/expert-spoon-ARCHIVED",
          },
        },
      ],
      "_type": "github_repo",
      "archived": false,
      "autoMergeAllowed": false,
      "createdOn": 1648058631000,
      "databaseId": 473303209,
      "deleteBranchOnMerge": false,
      "description": "Testing for archived repos",
      "disabled": false,
      "displayName": "expert-spoon-ARCHIVED",
      "empty": false,
      "fork": false,
      "forkCount": 0,
      "forkingAllowed": false,
      "fullName": "j1-ingest/expert-spoon-archived",
      "homepageUrl": "",
      "inOrganization": true,
      "lockReason": "",
      "locked": false,
      "mergeCommitAllowed": true,
      "mirror": false,
      "name": "expert-spoon-ARCHIVED",
      "node": "R_kgDOHDYIqQ",
      "owner": "j1-ingest",
      "public": false,
      "pushedOn": 1648058658000,
      "rebaseMergeAllowed": true,
      "securityPolicyEnabled": false,
      "template": false,
      "updatedOn": 1648058631000,
      "userConfigurationRepository": false,
      "webLink": "https://github.com/j1-ingest/expert-spoon-ARCHIVED",
    },
    Object {
      "_class": Array [
        "CodeRepo",
      ],
      "_key": "R_kgDOHXVLyg",
      "_rawData": Array [
        Object {
          "name": "default",
          "rawData": Object {
            "autoMergeAllowed": false,
            "createdAt": "2022-05-19T20:53:51Z",
            "databaseId": 494226378,
            "deleteBranchOnMerge": false,
            "description": "Forked to allow graph-github data ingestion testing",
            "forkCount": 0,
            "forkingAllowed": true,
            "homepageUrl": null,
            "id": "R_kgDOHXVLyg",
            "isArchived": false,
            "isDisabled": false,
            "isEmpty": false,
            "isFork": true,
            "isInOrganization": true,
            "isLocked": false,
            "isMirror": false,
            "isPrivate": false,
            "isSecurityPolicyEnabled": false,
            "isTemplate": false,
            "isUserConfigurationRepository": false,
            "lockReason": null,
            "mergeCommitAllowed": true,
            "name": "intentionally-vulnerable-golang-project",
            "nameWithOwner": "j1-ingest/intentionally-vulnerable-golang-project",
            "pushedAt": "2022-01-28T15:42:12Z",
            "rebaseMergeAllowed": true,
            "updatedAt": "2021-12-04T01:21:54Z",
            "url": "https://github.com/j1-ingest/intentionally-vulnerable-golang-project",
          },
        },
      ],
      "_type": "github_repo",
      "archived": false,
      "autoMergeAllowed": false,
      "createdOn": 1652993631000,
      "databaseId": 494226378,
      "deleteBranchOnMerge": false,
      "description": "Forked to allow graph-github data ingestion testing",
      "disabled": false,
      "displayName": "intentionally-vulnerable-golang-project",
      "empty": false,
      "fork": true,
      "forkCount": 0,
      "forkingAllowed": true,
      "fullName": "j1-ingest/intentionally-vulnerable-golang-project",
      "homepageUrl": "",
      "inOrganization": true,
      "lockReason": "",
      "locked": false,
      "mergeCommitAllowed": true,
      "mirror": false,
      "name": "intentionally-vulnerable-golang-project",
      "node": "R_kgDOHXVLyg",
      "owner": "j1-ingest",
      "public": true,
      "pushedOn": 1643384532000,
      "rebaseMergeAllowed": true,
      "securityPolicyEnabled": false,
      "template": false,
      "updatedOn": 1638580914000,
      "userConfigurationRepository": false,
      "webLink": "https://github.com/j1-ingest/intentionally-vulnerable-golang-project",
    },
  ],
  "collectedRelationships": Array [
    Object {
      "_class": "OWNS",
      "_fromEntityKey": "O_kgDOBfl5ZQ",
      "_key": "O_kgDOBfl5ZQ|owns|R_kgDOG5ypUQ",
      "_toEntityKey": "R_kgDOG5ypUQ",
      "_type": "github_account_owns_repo",
      "displayName": "OWNS",
    },
    Object {
      "_class": "OWNS",
      "_fromEntityKey": "O_kgDOBfl5ZQ",
      "_key": "O_kgDOBfl5ZQ|owns|R_kgDOG-zMhA",
      "_toEntityKey": "R_kgDOG-zMhA",
      "_type": "github_account_owns_repo",
      "displayName": "OWNS",
    },
    Object {
      "_class": "OWNS",
      "_fromEntityKey": "O_kgDOBfl5ZQ",
      "_key": "O_kgDOBfl5ZQ|owns|R_kgDOG_MJdQ",
      "_toEntityKey": "R_kgDOG_MJdQ",
      "_type": "github_account_owns_repo",
      "displayName": "OWNS",
    },
    Object {
      "_class": "OWNS",
      "_fromEntityKey": "O_kgDOBfl5ZQ",
      "_key": "O_kgDOBfl5ZQ|owns|R_kgDOG_MLeg",
      "_toEntityKey": "R_kgDOG_MLeg",
      "_type": "github_account_owns_repo",
      "displayName": "OWNS",
    },
    Object {
      "_class": "OWNS",
      "_fromEntityKey": "O_kgDOBfl5ZQ",
      "_key": "O_kgDOBfl5ZQ|owns|R_kgDOHDREFw",
      "_toEntityKey": "R_kgDOHDREFw",
      "_type": "github_account_owns_repo",
      "displayName": "OWNS",
    },
    Object {
      "_class": "OWNS",
      "_fromEntityKey": "O_kgDOBfl5ZQ",
      "_key": "O_kgDOBfl5ZQ|owns|R_kgDOHDYIqQ",
      "_toEntityKey": "R_kgDOHDYIqQ",
      "_type": "github_account_owns_repo",
      "displayName": "OWNS",
    },
    Object {
      "_class": "OWNS",
      "_fromEntityKey": "O_kgDOBfl5ZQ",
      "_key": "O_kgDOBfl5ZQ|owns|R_kgDOHXVLyg",
      "_toEntityKey": "R_kgDOHXVLyg",
      "_type": "github_account_owns_repo",
      "displayName": "OWNS",
    },
  ],
  "encounteredTypes": Array [
    "github_account",
    "github_repo",
    "github_account_owns_repo",
  ],
  "numCollectedEntities": 7,
  "numCollectedRelationships": 7,
}
`;
